### PR TITLE
Update guides and config reference - Closes #1278

### LINKED ISSUE #1278: Explain how to change the log level in Lisk Service
### Description
Improve explanations in guides and reference how to configure Lisk Service.

### Additional information

SERVICE_LOG_LEVEL: 'info'in the ecosystem config file (it's in there 3 times)

Given screenshot means your lisk-serviceis running using pm2. You can see the logs using pm2 logs process name . (For eg, pm2 logs lisk-service-core). Also, if you would like to start your lisk-service in docker, please follow: https://lisk.com/documentation/lisk-service/setup/docker.html---Thanks for this, but actually it will not work for my case as I need to create my own Dockerfile with lisk-service, so I guess the only way to do it is through binary setup?In current docker file I have entrypoint set as: exec pm2 start mainnet.config.jsnot sure if I can debug logs when container is exiting right after start 🤔---using docker logs you should be able to get startup error if printed to stdoutif not, you should look at pm2 log files location, and mount a local volume to that directory

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1278

### CODE REVIEW

File: modules/ROOT/pages/configuration/source.adoc [hunk 1]
```diff
@@ -16,24 +16,28 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 == The ecosystem config files
 
-The {url_pm2_ecosystem_file}[ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
+The ecosystem file^]
```
Comment (sameersubudhi) [inline modules/ROOT/pages/configuration/source.adoc]: Did you intend to remove this line?

File: modules/ROOT/pages/configuration/source.adoc [hunk 2]
```diff
@@ -16,24 +16,28 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 ----
-pm2 start ecosystem.mainnet.config.js
+
 ----
 
 === Default configurations
 
-The default ecosystem file `ecosystem.core3.config.js` is located in the root folder of `lisk-service`:
+The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
+The default confoguration file is `ecosystem.core3.config.js`, located in the root folder of `lisk-service`:
```
Comment (sameersubudhi) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The default configuration file is `ecosystem.core3.config.js`, located in the root directory of `lisk-service`:
```

File: modules/ROOT/pages/configuration/source.adoc [hunk 3]
```diff
@@ -16,24 +16,28 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 It contains a configuration to connect to a local Lisk Core node.
 
 This config will be used by default when starting Lisk Service with the following:
 
 [source,bash]
 ----
 pm2 start
+
+# or
+# pm2 start ecosystem.core3.config.js
```
Comment (sameersubudhi) [inline modules/ROOT/pages/configuration/source.adoc]: Kindly consider updating it to the following as we do not have a default `ecosystem.config.js` file.
```suggestion
pm2 start ecosystem.core3.config.js
```

File: modules/ROOT/pages/management/source.adoc [hunk 4]
```diff
@@ -62,3 +57,14 @@ npm stop
...
+== Logging
+
+To check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container you want to see the logs of.
+
+For example, to see the logs for the Gateway microservice, execute:
+
+[source,bash]
+----
+docker container logs lisk-service_gateway_1
+----
```
Comment (sameersubudhi) [inline modules/ROOT/pages/management/source.adoc]: Kindly update
```suggestion
To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2 managed process you want to see the logs of.

For example, to see the logs for the Gateway microservice, execute:

[source,bash]
----
pm2 logs lisk-service-gateway
----
```

File: modules/ROOT/pages/references/configuration.adoc [hunk 5]
```diff
@@ -18,15 +14,15 @@ If the Docker environment is being used, only the required variable `LISK_CORE_W
 SERVICE_BROKER=redis://localhost:6379/0
 ----
 
-=== Log configuration
+=== Logging configuration
 
 [source,bash]
 ----
+SERVICE_LOG_LEVEL=info    # Default log level. TRACE < DEBUG < INFO < WARN < ERROR < FATAL < MARK
```
Comment (sameersubudhi) [inline modules/ROOT/pages/references/configuration.adoc]: `FATAL` is the logging level with the highest severity.

```suggestion
SERVICE_LOG_LEVEL=info    # Default log level. TRACE < DEBUG < INFO < WARN < ERROR < FATAL
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Please find the following observations.

File: modules/ROOT/pages/configuration/source.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/configuration/source.adoc]: Yes you are right! Removed now :)

**Review by sameersubudhi — DISMISSED** [review]
Thank you for the changes! LGTM 👍

File: modules/ROOT/pages/configuration/source.adoc [hunk 6]
```diff
@@ -14,26 +14,17 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-.Starts Lisk Service with the specified configuration
-[source,bash]
-----
-pm2 start ecosystem.mainnet.config.js
-----
-
-=== Default configurations
-
-The default ecosystem file `ecosystem.core3.config.js` is located in the root folder of `lisk-service`:
+The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
```
Comment (chris529) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
```

Comment (chris529) [reply modules/ROOT/pages/configuration/source.adoc]: Hey Mona, I think the latter part of this statement should read as "the **log** files of Lisk Service with PM2" Thoughts...?

File: modules/ROOT/pages/configuration/source.adoc [hunk 7]
```diff
@@ -14,26 +14,17 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+The default configuration file is `ecosystem.core3.config.js`, located in the root folder of `lisk-service`:
```
Comment (chris529) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The default configuration file is `ecosystem.core3.config.js`, and is located in the root folder of `lisk-service`:
```

File: modules/ROOT/pages/configuration/source.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/configuration/source.adoc]: Can't really spot it - what is the difference here?

File: modules/ROOT/pages/management/docker.adoc [hunk 8]
```diff
@@ -31,3 +31,21 @@ make down
...
+== Show status of Docker containers
+
+[source,bash]
+----
+docker ps
+----
+
+== Logging
+
+To check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container you want to see the logs of.
```
Comment (chris529) [inline modules/ROOT/pages/management/docker.adoc]:
```suggestion
To check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container that holds the logs you wish to view.
```

File: modules/ROOT/pages/management/source.adoc [hunk 9]
```diff
@@ -62,3 +57,14 @@ npm stop
 ----
 make clean
 ----
+
+== Logging
+
+To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2 managed process you want to see the logs of.
```
Comment (chris529) [inline modules/ROOT/pages/management/source.adoc]:
```suggestion
To check the logs for the different microservices of Lisk Service, use the command `pm2 logs PROCESS_NAME`, where `PROCESS_NAME` is the respective PM2 managed process that contains the logs you wish to view.
```

File: modules/ROOT/pages/references/configuration.adoc [hunk 10]
```diff
@@ -68,6 +64,35 @@ ENABLE_HTTP_CACHE_CONTROL=true
...
+# variable is required to be true.
+ENABLE_REQUEST_CACHING=true
+----
+
+=== Websocket settings
+
+[source,bash]
+----
+# To enable websocket rate limit, the `WS_RATE_LIMIT_ENABLE` environment variable is required to be true.
+# Number of connections per second can be set using
```
Comment (chris529) [inline modules/ROOT/pages/references/configuration.adoc]:
```suggestion
# Number of connections per second can be set using the
```

File: modules/ROOT/pages/references/configuration.adoc [hunk 11]
```diff
@@ -68,6 +64,35 @@ ENABLE_HTTP_CACHE_CONTROL=true
...
+# `WS_RATE_LIMIT_CONNECTIONS` and `WS_RATE_LIMIT_DURATION` environment
+# variable. currently defaults to `5 connections per second`
```
Comment (chris529) [inline modules/ROOT/pages/references/configuration.adoc]:
```suggestion
# variable. It currently defaults to `5 connections per second`
```

File: modules/ROOT/pages/references/configuration.adoc [hunk 12]
```diff
@@ -68,6 +64,35 @@ ENABLE_HTTP_CACHE_CONTROL=true
...
+WS_RATE_LIMIT_CONNECTIONS=5
+WS_RATE_LIMIT_DURATION=1
+----
+
+=== Compatibility settings
+
+[source,bash]
+----
+# To enable `HTTP Rate limit`, the `HTTP_RATE_LIMIT_ENABLE` environment variable is required to be true.
+# The `HTTP Rate limit` directives can be set using `HTTP_RATE_LIMIT_WINDOW` and `HTTP_RATE_LIMIT_CONNECTIONS` environment
```
Comment (chris529) [inline modules/ROOT/pages/references/configuration.adoc]:
```suggestion
# The `HTTP Rate limit` directives can be set using the `HTTP_RATE_LIMIT_WINDOW` and the `HTTP_RATE_LIMIT_CONNECTIONS` environment
```

File: modules/ROOT/pages/references/configuration.adoc [hunk 13]
```diff
@@ -68,6 +64,35 @@ ENABLE_HTTP_CACHE_CONTROL=true
...
+# variable.
+HTTP_RATE_LIMIT_ENABLE=true
+HTTP_RATE_LIMIT_WINDOW=10 # To keep record of requests in memory (in seconds). Defaults to 10 seconds
```
Comment (chris529) [inline modules/ROOT/pages/references/configuration.adoc]:
```suggestion
HTTP_RATE_LIMIT_WINDOW=10 # To keep a record of requests in the memory (in seconds). Defaults to 10 seconds
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor point 👍

File: modules/ROOT/pages/configuration/source.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/configuration/source.adoc]: Sorry, I should have written the entire sentence..here it is:
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the log files of Lisk Service with PM2.             (Not logs files)

Comment (Tschakki) [reply modules/ROOT/pages/configuration/source.adoc]: ah, I see, yes you are right :)

Comment (Tschakki) [reply modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the log files of Lisk Service with PM2.
```

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Pull request merged**